changelog.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Add tags list to filter project.
   * Add search bar for projects.
   * Add search bar for templates.
+* Multi diagrams page:
+  * Display all diagrams of a project.
+  * Zoom center on all diagrams.
```

Comment:
`Zoom to center of all diagrams.` OR `Zoom centered on all diagrams.`

Reply:
Done